schema/xml/metaschema.xsd

```diff
@@ -739,7 +740,7 @@
       <xs:extension base="MatchesConstraintType">
         <xs:attribute name="target" type="MetaschemaPathType" use="required">
           <xs:annotation>
-            <xs:documentation>Specifies the target of the constraint as a metaschema path. If the value is "." and the containing metaschema object is a field, the constraint applies to the field's value. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
+            <xs:documentation>Specifies the target of the constraint as a Metapath. If the value is "." and the containing Metaschema object is a field, the constraint applies to the field's value. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
```

Comment:
```suggestion
            <xs:documentation>Specifies the target of the constraint as a Metapath expression. If the value is "." and the containing Metaschema object is a field, the constraint applies to the field's value. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
```